src/Helpers/helpers.js

```diff
@@ -14,14 +14,20 @@ export function cmpStr (str1, str2) {
 
 /**
  * capitalize all first letters of sentence
- * @param str
+ * @param words
  * @returns {string}
  */
-export function ucFirstAllWords (str) {
-	var pieces = str.split(" ");
-	for (var i = 0; i < pieces.length; i++) {
-		var j = pieces[i].charAt(0).toUpperCase();
-		pieces[i] = j + pieces[i].substr(1);
+function capitalizeFirstLetter (sentence) {
+	const words = sentence.split(" ");
+	for (var i = 0; i < words.length; i++) {
+		var j = words[i].charAt(0).toUpperCase();
+		words[i] = j + words[i].substr(1);
 	}
-	return pieces.join(" ");
+	return words.join(" ");
 }
+
+
+export default {
+	compareString,
+	capitalizeFirstLetter
+}
```

Comment:
Out of scope but I think it's a good discussion point:
IMO we should move away from default exports as they may become a pain point when refactoring code.